app/controllers/image_sets_controller.rb

```diff
@@ -56,7 +56,7 @@ def image_set_params
           :url,
           :lion_id,
           :user_id,
-          :age,
+          :lion_date_of_birth,
```

Comment:
I would call this `date_of_birth` instead of `lion_date_of_birth` since we don't prefix the other properties (ie. name and gender) with `lion`